TlmCmd/common_cmd_packet_util.c

```diff
@@ -113,46 +157,93 @@ CCP_UTIL_ACK CCP_form_block_deploy_cmd(CommonCmdPacket* packet, TLCD_ID tl_no, b
   return CCP_form_rtc(packet, Cmd_CODE_TLCD_DEPLOY_BLOCK, param, 1 + SIZE_OF_BCT_ID_T);
 }
 
-PH_ACK CCP_register_app_cmd(cycle_t ti, AR_APP_ID id)
+void CCP_convert_rtc_to_tlc(CommonCmdPacket* packet, cycle_t ti)
 {
-  CCP_form_app_cmd(CCP_util_packet_, ti, id);
-  return PH_analyze_cmd_packet(CCP_util_packet_);
+  CCP_set_exec_type(packet, CCP_EXEC_TYPE_TL_FROM_GS);
+  CCP_set_ti(packet, ti);
+}
+
+PH_ACK CCP_register_app_cmd(cycle_t ti, CCP_EXEC_TYPE type, AR_APP_ID id)
+{
+  CCP_form_app_cmd(&CCP_util_packet_, ti, type, id);
+  return PH_analyze_cmd_packet(&CCP_util_packet_);
 }
```

Comment:
[IMO]

上の議論で，鈴本の意見だと，ここはこうなる？

```suggestion
PH_ACK CCP_register_app_cmd(cycle_t ti, CCP_EXEC_TYPE type, AR_APP_ID id)
{
  CCP_form_app_cmd(&CCP_util_packet_, ti, id);
  CCP_set_exec_type(packet, type);
  return PH_analyze_cmd_packet(&CCP_util_packet_);
}
```

Reply:
下と共通しますが、 app_cmd と deploy の register は消しました